common/locales/en/merch.json

```diff
@@ -0,0 +1,14 @@
+{
+  "merch" : "Merch",
```

Comment:
Although probably wait until you hear more feedback so you can do it all at once. :)